src/lib/rule-finder.js

```diff
@@ -108,6 +108,6 @@ function RuleFinder(specifiedFile, options) {
   this.getUnusedRules = () => getSortedRules(unusedRules);
 }
 
-module.exports = function (specifiedFile, options = {}) {
+module.exports = function (specifiedFile, options) {
```

Comment:
Then let's go with specifying the default value in the exported function... the original `RuleFinder` function is "private" either way.

I will do this within the next few hours, when preparing a `v3.0.0` release, if no new arguments against it arise until then.